src/IndexedDbProvider.ts

```diff
@@ -483,20 +480,20 @@ class IndexedDbStore implements NoSqlProvider.DbStore {
     }
 
     getMultiple(keyOrKeys: KeyType|KeyType[]): SyncTasks.Promise<ItemType[]> {
-        let keys: any[];
-        const err = _.attempt(() => {
-            keys = NoSqlProviderUtils.formListOfKeys(keyOrKeys, this._schema.primaryKeyPath);
+        const keys = _.attempt(() => {
+            const keys = NoSqlProviderUtils.formListOfKeys(keyOrKeys, this._schema.primaryKeyPath);
```

Comment:
It looks like `keys` is only needed in the `else` case. Why is it computed up here and then thrown away in the `if` case? Or does this have a side-effect?

[Same in the next `_.attempt`.]

Reply:
It's used in both cases 
`_.map(keys, key => NoSql`